### PR TITLE
Prep of 0.14 release

### PR DESCRIPTION
**Pull request by veronicagg**
- Made separate commits for generated code for gems
- One commit with all the manual files for the new gems
- One commit with azure_sdk updates, gemfile, rakefile, swagger to sdk and readme

Please review and point out if anything is missing. 
Generation was done with autorest code 2.0.4147 and autorest.ruby 2.0.15.
Addressing https://github.com/Azure/azure-sdk-for-ruby/issues/1024

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -169,11 +179,26 @@ REGEN_METADATA = {
...
+        version: version
+    },
     azure_mgmt_container_registry: {
         spec_uri: 'https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/containerregistry/resource-manager/readme.md',
         ns: 'Azure::ARM::ContainerRegistry',
         version: version
     },
+    azure_mgmt_customer_insights: {
+        spec_uri: 'https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/customer-insights/resource-manager/readme.md',
+        ns: 'Azure::ARM::ContainerInsights',
```
Comment (vishrutshah) [inline Rakefile]: Should `Azure::ARM::ContainerInsights` be `Azure::ARM::CustomerInsights` ?

File: swagger_to_sdk_config.json [hunk 2]
```diff
@@ -105,6 +153,18 @@
         "azure_mgmt_container_registry.rb"
       ]
     },
+    "azure_mgmt_customer_insights": {
+      "markdown": "specification/customer-insights/resource-manager/readme.md",
+      "autorest_options": {
+        "namespace": "Azure::ARM::ContainerInsights",
```
Comment (vishrutshah) [inline swagger_to_sdk_config.json]: `Azure::ARM::CustomerInsights` ?

File: swagger_to_sdk_config.json [hunk 3]
```diff
@@ -33,6 +33,18 @@
         "azure_mgmt_authorization.rb"
       ]
     },
+    "azure_mgmt_automation": {
+      "markdown": "specification/automation/resource-manager/readme.md",
+      "autorest_options": {
+        "namespace": "Azure::ARM::Automation",
+        "package-name": "azure_mgmt_automation",
+        "package-version": "0.13.0"
```
Comment (vishrutshah) [inline swagger_to_sdk_config.json]: Should this be `0.14.0` now ?

File: azure_sdk/azure_sdk.gemspec [hunk 4]
```diff
@@ -32,40 +32,55 @@ Gem::Specification.new do |spec|
...
+  spec.add_runtime_dependency 'azure_mgmt_relay', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_resources', "~>#{version}"
+  spec.add_runtime_dependency 'azure_mgmt_resources_management', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_scheduler', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_search', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_server_management', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_service_bus', "~>#{version}"
+  spec.add_runtime_dependency 'azure_mgmt_service_fabric', "~>#{version}"
   spec.add_runtime_dependency 'azure_mgmt_sql', "~>#{version}"
+  spec.add_runtime_dependency 'azure_mgmt_stor_simple8000series', "~>#{version}"
```
Comment (vishrutshah) [inline azure_sdk/azure_sdk.gemspec]: `azure-storage` version [0.14.0.preview](https://rubygems.org/gems/azure-storage/versions/0.14.0.preview) has been released. We should update to that as usual. :)

File: Rakefile (hunk 1 above)
Comment (veronicagg) [reply Rakefile]: Thanks for catching it, this one I addressed with a later commit, so it should be good in latest.

File: swagger_to_sdk_config.json (hunk 3 above)
Comment (veronicagg) [reply swagger_to_sdk_config.json]: Updating this.

File: swagger_to_sdk_config.json (hunk 2 above)
Comment (veronicagg) [reply swagger_to_sdk_config.json]: Thanks for catching, updating this.

File: azure_sdk/azure_sdk.gemspec (hunk 4 above)
Comment (veronicagg) [reply azure_sdk/azure_sdk.gemspec]: Updating this, thanks!

**Comment by vishrutshah** [comment]
@veronicagg : FYI @sarangan12 mentioned about the `azure_mgmt_monitor` bug and that we may want it to go with this release or not. Feel free to sync up before pushing `release` button :)

**Comment by veronicagg** [comment]
@vishrutshah we'll release as is, we'll make the update for monitor in the next one. thanks!!

**Pull request merged**